services/datamanager/builtin/builtin.go

```diff
@@ -310,28 +305,47 @@ func (svc *builtIn) closeSyncer() {
 		svc.syncer.Close()
 		svc.syncer = nil
 	}
+	if svc.cloudConn != nil {
+		goutils.UncheckedError(svc.cloudConn.Close())
+	}
 }
 
-func (svc *builtIn) initSyncer(cfg *config.Config) error {
-	syncer, err := svc.syncerConstructor(svc.logger, cfg)
+var grpcConnectionTimeout = 10 * time.Second
+
+func (svc *builtIn) initSyncer(ctx context.Context) error {
+	ctx, cancel := context.WithTimeout(ctx, grpcConnectionTimeout)
+	defer cancel()
+
+	identity, conn, err := svc.cloudConnSvc.AcquireConnection(ctx)
+	if errors.Is(err, cloud.ErrNotCloudManaged) {
+		svc.logger.Debug("Using no-op sync manager when not cloud managed")
+		svc.syncer = datasync.NewNoopManager()
+	}
+	if err != nil {
+		return err
+	}
+
+	client := v1.NewDataSyncServiceClient(conn)
+
+	syncer, err := svc.syncerConstructor(identity, client, svc.logger)
 	if err != nil {
 		return errors.Wrap(err, "failed to initialize new syncer")
 	}
 	svc.syncer = syncer
+	svc.cloudConn = conn
 	return nil
 }
 
 // getCollectorFromConfig returns the collector and metadata that is referenced based on specific config atrributes
-func (svc *builtIn) getCollectorFromConfig(attributes dataCaptureConfig) (data.Collector, *componentMethodMetadata) {
+func (svc *builtIn) getCollectorFromConfig(attributes datamanager.DataCaptureConfig) (data.Collector, *componentMethodMetadata) {
 	// Create component/method metadata to check if the collector exists.
 	metadata := data.MethodMetadata{
-		Subtype:    attributes.Type,
+		Subtype:    attributes.Resource.Name().Subtype,
 		MethodName: attributes.Method,
 	}
 
 	componentMetadata := componentMethodMetadata{
-		ComponentName:  attributes.Name,
-		ComponentModel: attributes.Model,
```

Comment:
correct. I thought that app already stopped caring if those were set or not

Reply:
Yep, app now passes model through if it's set but does not require it. Since we're fully removing from RDK, the above ticket will fully remove from app (don't include in filter, etc)